packages/keystone/logging/GraphQLLoggerApp.js

```diff
@@ -39,32 +39,50 @@ function renderExecutableDefinitionNode (node) {
     return ''
 }
 
+/**
+ * Get current time in nanoseconds and return diff between
+ * @param time {bigint} start time of the operation
+ * @returns {number} Operation elapsed time in milliseconds
+ */
+function endTimer (time) {
+    function roundTo (decimalPlaces, numberToRound) {
```

Comment:
We can use native JS for this :)
<img width="576" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/ea9d874a-37c6-45cb-8476-18d9aa615b6d">

Reply:
Good suggestion, I'll refactor it with `toFixed`